app/src/main/java/com/nilhcem/blenamebadge/ui/badge_preview/PreviewBadge.kt

```diff
@@ -95,30 +108,216 @@ class PreviewBadge : View {
         // Draw Cells
         for (i in 0 until badgeHeight) {
             for (j in 0 until badgeWidth) {
-                if (i < checkList?.size ?: -1 && j < checkList?.get(i)?.list?.size ?: -1 && checkList?.get(i)?.list?.get(j) == true) {
-                    ledEnabled?.bounds = cells[i].list[j]
-                    ledEnabled?.draw(canvas)
+                if (ifFlash == true) {
```

Comment:
Please remove `if (ifFlash == true)`, `if (ifFlash)` is enough.

Reply:
ifFlash is a nullable property, it needs to be checked against its value.